### PR TITLE
[SPARK-19330][DStreams] Also show tooltip for successful batches

### PR DESCRIPTION
**Pull request by lw-lin**
### Before
_streaming_before

### After
_streaming_after

## How was this patch tested?

Manually

### CODE REVIEW

File: streaming/src/main/resources/org/apache/spark/streaming/ui/static/streaming-page.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ function drawTimeline(id, data, minX, maxX, minY, maxY, unitY, batchInterval) {
             .style("cursor", "pointer")
             .attr("cx", function(d) { return x(d.x); })
             .attr("cy", function(d) { return y(d.y); })
-            .attr("r", function(d) { return isFailedBatch(d.x) ? "2" : "0";})
+            .attr("r", function(d) { return isFailedBatch(d.x) ? "2" : "3";})
```
Comment (srowen) [inline streaming/src/main/resources/org/apache/spark/streaming/ui/static/streaming-page.js]: What does this do?

**Comment by uncleGen** [comment]
Sorry, I do not get your point. Do you mean the precessing time detail can not be shown on line? I can not reproduce it, and everything works well.

File: streaming/src/main/resources/org/apache/spark/streaming/ui/static/streaming-page.js (hunk 1 above)
Comment (lw-lin) [inline streaming/src/main/resources/org/apache/spark/streaming/ui/static/streaming-page.js]: This changes the radius of a blue circle (which represents a successful batch) in the svg.

Prior to this patch, the radius was 0 thus the blue circle could not be hovered and tooltip would not show. This patch alters the radius to 3 for each blue circle.

**Comment by lw-lin** [comment]
@uncleGen yea I mean the tooltip won't show for a blue circle (which represents a successful batch) when the cursor hovers over the blue circle in the 'processing time' line. Which version of spark are you running?

**Comment by lw-lin** [comment]
@zsxwing would you take a look, thanks!

**Comment by zsxwing** [comment]
Yeah, much easier to see the batch tooltip after this change. LGTM. Thanks! Merging to master and 2.1.

**Pull request closed without merging**